docs/UserGuide.md

```diff
@@ -125,6 +128,18 @@ This guide serves to:<br>
 * An example is shown below:
 ![SortExample](images/SortExample.png)
 
+#### Responsive Ui when window resizing
+
+* The Ui will change dynamically when resized.
```

Comment:
I think "Ui" should be standardised to "UI" throughout the UG, since its an acronym?

Reply:
Done, all references to Ui were changed to UI.